test/public/test/call-remote-callbacks.js

```diff
@@ -275,6 +275,29 @@ asyncTest('required radio should only require one to be checked', 1, function()
   }, 13);
 });
 
+asyncTest('required radio should only require one to be checked if not all radios are required', 1, function() {
+  $(document).bind('iframe:loading', function() {
```

Comment:
> Anyway, again, I don't really care, as on seems to have won out. Just wanted to point out that you probably shouldn't replace them due to being deprecated, since they're not.

:+1:. Both are fine to me so it is better to keep using `bind/delegate`